docs/app-integration-development/20-OAuth-2-Extended-OAuth.md

```diff
@@ -0,0 +1,67 @@
+---
+tags: [app-integration-development]
+---
+
+# Extended OAuth
+
+<!-- theme: warning -->
+> ### Early Access
+> 
+> The features described on this page are in an Early Access state and are subject to change. Your PagerDuty Account may 
+> require a feature flag before this functionality is available to you. Please reach out to us if you have any questions or 
+> need support.
+
+## Register an App
```

Comment:
There is already a page called "Register an App" we should have that page linked to this too once we are in EA.